.github/ISSUE_TEMPLATE/question.md

```diff
@@ -0,0 +1,10 @@
+---
+name: 💬 Question
+about: If you need help understanding HumanConnection.
```

Comment:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-04-02T15:22:03Z" title="Tuesday, April 2nd 2019, 5:22:03 pm +02:00">Apr 2, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -0,0 +1,10 @@
+---
+name: 💬 Question
+about: If you need help understanding the HumanConnection.
```


```suggestion
about: If you need help to understand Human Connection.
```

Reply:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2019-04-02T18:41:16Z" title="Tuesday, April 2nd 2019, 8:41:16 pm +02:00">Apr 2, 2019</time>_

---


Actually I think grammatically it's better `need help understanding`, but I would also suggest to lose `the` 
Suggested: `If you need help understanding Human Connection`

Reply:
> [<img alt="ulfgebhardt" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/ulfgebhardt) **Authored by [ulfgebhardt](https://github.com/ulfgebhardt)**
_<time datetime="2019-04-02T19:09:05Z" title="Tuesday, April 2nd 2019, 9:09:05 pm +02:00">Apr 2, 2019</time>_

---


will adopt!